src/components/icons/svg/ProgressIcon.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import Animated, { useAnimatedProps } from 'react-native-reanimated';
+import { G, Path } from 'react-native-svg';
+import { Centered } from '../../layout';
+import Svg from '../Svg';
+import { useTheme } from '@rainbow-me/context';
+import { position } from '@rainbow-me/styles';
+
+const AnimatedSvg = Animated.createAnimatedComponent(Svg);
+const AnimatedPath = Animated.createAnimatedComponent(Path);
+
+const convertProgress = (progress: number) => {
+  'worklet';
+  return (360 * Math.min(100, Math.max(0, progress))) / 100;
+};
```

Comment:
Do you use values 0 to 1 that you convert to degrees and then convert degrees to radians? Can't you just use radians directly?